src/azure-cli/azure/cli/command_modules/backup/commands.py

```diff
@@ -7,72 +7,90 @@
+
     backup_vaults_sdk = CliCommandType(
         operations_tmpl='azure.mgmt.recoveryservices.operations#VaultsOperations.{}',
         client_factory=vaults_cf)
 
     backup_storage_config_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.recoveryservices.operations#BackupStorageConfigsOperations.{}',
+        operations_tmpl='azure.mgmt.recoveryservicesbackup.operations.backup_resource_storage_configs_operations#BackupResourceStorageConfigsOperations.{}',
         client_factory=vaults_cf)
 
     with self.command_group('backup vault', backup_vaults_sdk, client_factory=vaults_cf) as g:
         g.custom_command('create', 'create_vault')
         g.show_command('show', 'get')
         g.custom_command('list', 'list_vaults')
-        g.show_command('backup-properties show', 'get', command_type=backup_storage_config_sdk)
+        g.show_command('backup-properties show', 'get', command_type=backup_storage_config_sdk, client_factory=backup_storage_configs_cf)
         g.custom_command('backup-properties set', 'set_backup_properties', client_factory=backup_storage_configs_cf)
         g.custom_command('delete', 'delete_vault', confirmation=True)
 
-    with self.command_group('backup container', backup_custom, client_factory=backup_protection_containers_cf) as g:
-        g.show_command('show', 'show_container')
-        g.command('list', 'list_containers', table_transformer=transform_container_list)
+    with self.command_group('backup container', backup_custom_base, client_factory=protection_containers_cf) as g:
+        g.show_command('show', 'show_container', client_factory=backup_protection_containers_cf)
+        g.command('list', 'list_containers', table_transformer=transform_containers_list, client_factory=backup_protection_containers_cf)
+        g.command('register', 'register_wl_container')
+        g.command('re-register', 're_register_wl_container', confirmation=True)
+        g.command('unregister', 'unregister_wl_container', confirmation=True)
 
-    with self.command_group('backup policy', backup_custom, client_factory=protection_policies_cf) as g:
+    with self.command_group('backup policy', backup_custom_base, client_factory=protection_policies_cf) as g:
         g.command('get-default-for-vm', 'get_default_policy_for_vm')
-        g.show_command('show', 'show_policy')
-        g.command('list', 'list_policies', client_factory=backup_policies_cf, table_transformer=transform_policy_list)
+        g.show_command('show', 'show_policy', table_transformer=transform_wl_policy_show)
+        g.command('list', 'list_policies', client_factory=backup_policies_cf, table_transformer=transform_policies_list)
         g.command('list-associated-items', 'list_associated_items_for_policy', client_factory=backup_protected_items_cf, table_transformer=transform_item_list)
-        g.command('set', 'set_policy')
+        g.command('set', 'set_policy', table_transformer=transform_wl_policy_set)
```

Comment:
do we need a table transformer for set policy?

Reply:
Yes given in spec